homeworks/HW1/src/InputValidatedField.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Input} from 'reactstrap';
+import './App.css';
+
+export class InputValidatedField extends React.Component {
+
+    constructor(props) {
+        super(props);
+        const error = {
+            message: null
+        };
+        this.state = {
+            error: error
+        }
+        this.validateInput = this.validateInput.bind(this);
+    }
+
+    validateInput = (e) => {
+        const input = e.target.value;
```

Comment:
That is not `input` that is a value from an input.
Please name var to more clarified.